TwoPaneLayout/README.md

```diff
@@ -2,7 +2,9 @@
 
 **TwoPaneLayout** is a UI component for Jetpack Compose, which contains the layouts that help you create UI for dual-screen, foldable, and large-screen devices. TwoPaneLayout provides a two-pane layout for use at the top level of a UI. The component will place two panes side-by-side when the app is spanned on dual-screen, foldable and large-screen devices. The two panes can be horizontal or vertical, based on the orientation of the device, unless `paneMode` is configured. 
 
-When the app is spanned across a vertical hinge or fold, or when the width is larger than height of screen on large-screen device, pane1 will be placed on the left, while pane2 will be on the right. If the device rotates, the app is spanned across a horizontal hinge or fold, or the width is smaller than the height of screen on large-screen device, pane1 will be placed on the top and pane2 will be on the bottom. The TwoPaneLayout is able to assign children widths or heights according to their weights provided using the `TwoPaneScope.weight` modifier. If no weight is provided, the two panes will be displayed equally.
+When the app is spanned across a vertical hinge or fold, or when the width is larger than height of screen on large-screen device, pane1 will be placed on the left, while pane2 will be on the right. If the device rotates, the app is spanned across a horizontal hinge or fold, or the width is smaller than the height of screen on large-screen device, pane1 will be placed on the top and pane2 will be on the bottom. 
+
+The TwoPaneLayout is able to assign children widths or heights according to their weights provided using the `TwoPaneScope.weight` modifier. If no weight is provided, the two panes will be displayed equally. The proportional layout only applies when the foldingFeature is not separating, meaning the foldable device is fully open or running on the large screen device. For dual-screen device with a hinge in the middle, the two panes would always layout equally.
```

Comment:
I think we could add a little more here to make sure it's clear how the pane sizes are determined! From what I understand, maybe this description could work (please feel free to edit, I wasn't sure how to best describe each case):

The TwoPaneLayout is able to assign children widths or heights according to their weights provided using the `TwoPaneScope.weight` modifier.
- For large screens, if no weight is provided, the two panes will be displayed equally. If weight is provided, then the layout will be split up proportionally according to the weight.
- For foldables with a separating folding feature, the two panes will always be split up according to the folding feature boundaries (with or without weight). A folding feature is considered separating when a device is half-opened or has a hinge that physically divides the screens. If a foldable device does not have a separating folding feature, then it will either be treated as a large screen or a single screen depending on its size.